LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Ketheroth
```

Comment:
Wrong year, also this should include the current licensing for the assets as those are licensed differently.